### ISSUE TITLE
I don't know why val obj_loss↑ 

### ISSUE THREAD

**Issue by foge1993** [ultralytics/yolov5#1999]
## ❔Question
I was training custom datasets, but why didn't the val objectness decline.   
![image](https://user-images.githubusercontent.com/37509794/105271076-d8e34e80-5bd1-11eb-961f-8cf61749b5ee.png)

**Comment by foge1993** [comment]
My datasets have 309 images with 10 classes. Could someone know the reasons?

**Comment by glenn-jocher** [comment]
@foge1993 your val losses do decrease. 

Overfitting val objectness loss is present prior to overfitting on your other components. Ideally you'd want your 3 loss components to begin overfitting at similar points in time. That said, its up to you to experiment with ways to improve this, such as using different hyperparameters, lr schedulers, models, increasing your dataset obviously also if you only have 300 images, etc.
